packages/jest-runtime/src/index.ts

```diff
@@ -520,6 +520,11 @@ export default class Runtime {
             initializeImportMeta: (meta: JestImportMeta) => {
               meta.url = pathToFileURL(modulePath).href;
 
+              // @ts-expect-error Jest uses @types/node@16. Will be fixed when updated to @types/node@20.11.0
+              meta.filename = fileURLToPath(meta.url);
```

Comment:
should we only add these if the underlying Node version supports them?

Reply:
I would say it should not be necessary as users on node < 20.11 won't use those variables in code.

Reply:
@SimenB Do you want to add the check? I would backport this change also to v29 when it is ready.

Reply:
>Do you want to add the check?

nah, I'm fine with it. If a feature check were easy it'd be fine, but it's a bit clunky, and I don't wanna check against specific node versions.

>I would backport this change also to v29 when it is ready.

We don't do backports, so there's no need for that. Thanks for offering, though!